keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -440,6 +451,127 @@ set of candidates and selected strategy is the same as before.
 The obvious drawback is the need for a consensus protocol and extra information
 in the `LeaseCandidate` object that may be unnecessary.
 
+### Priority-based Coordinated Leader Election
+To enhance control over leader assignment beyond existing CLE strategies like OldestEmulatedVersion, we propose adding an optional `Priority` field (an int32 with 0 as default, higher value = higher priority) to `LeaseCandidateSpec`. This field allows operators to explicitly designate a preferred leader. The CLE system will select the candidate with the highest non-zero Priority. If multiple candidates share the same highest priority, the existing v1.CoordinatedLeaseStrategy will act as a tie-breaker. If no candidates have a priority set, the system defaults to the existing v1.CoordinatedLeaseStrategy. This provides granular, temporary control without replacing the primary CLE mechanism.
+
+#### Proto Update
+The `LeaseCandidateSpec` will be updated as follows:
+```go
+// LeaseCandidateSpec is a specification of a Lease.
+type LeaseCandidateSpec struct {
```

Comment:
It's probably fine to drop the documentation for the other fields so the reader can more easily see the priority field and our changes.